src/views/tutorial-library/components/pagination/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import ButtonLink from 'components/button-link'
+import Button from 'components/button'
+import { usePagination } from 'react-instantsearch-hooks-web'
+
+import paginationStyle from './pagination.module.css'
+import { ButtonLinkProps } from 'components/button-link/types'
+
+/**
+ * @TODO de-duplicate some of the link generation, especially on either side of the pages array
+ */
```

Comment:
Another TODO I think we need here is regarding focus management. Activating one of the `ButtonLink`s visually moves to the top of the page, but the focus stays on the element that was activated.

Reply:
Great call!

Reply:
Could you add the TODO? Apologies for not making that request clearer 😊